src/components/data/data-utils.js

```diff
@@ -647,7 +647,7 @@ const AttributeInput = ({ value: attributeValue, onChange, entityTypes = [] }) =
           }),
           h(Link, {
             'aria-label': `Remove list value ${i + 1}`,
-            disabled: value.length === 1,
+            disabled: attributeValue.items.length === 1,
```

Comment:
It would be nice to use Lodash, in this case for consistency.
```suggestion
            disabled: _.size(attributeValue.items) === 1,
```